Basic-Car-Maintenance/Shared/Info/InfoView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  InfoView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import SwiftUI
+
+struct InfoView<ViewModel: InfoViewModel>: View {
```

Comment:
Use a different name, this is not really specific and doesn't really describe what the view is. Also remove the constraint that the view model needs to be an `InfoViewModel`